resources/lrsql/config/prod/default/webserver.edn

```diff
@@ -13,4 +13,7 @@
  :ssl-port          #long #or [#env LRSQL_SSL_PORT 8443]
  :url-prefix        #or [#env LRSQL_URL_PREFIX "/xapi"]
  :enable-admin-ui   #boolean #or [#env LRSQL_ENABLE_ADMIN_UI true]
- :enable-stmt-html  #boolean #or [#env LRSQL_ENABLE_STMT_HTML true]}
+ :enable-stmt-html  #boolean #or [#env LRSQL_ENABLE_STMT_HTML true]
+ :openid-issuer     #env LRSQL_OPENID_ISSUER
+ :openid-config     #env LRSQL_OPENID_CONFIG
+ :jwks-uri          #env LRSQL_JWKS_URI}
```

Comment:
Hmm I guess I never realized you could use `#env` without `#or`. Do these values get assigned `nil` or do they just not appear in the config map (which would have implications for the config spec)?

Reply:
It was my impression that they come through as nil, but then a function cleans out nils. I'll double check that now

Reply:
Oh yeah I forgot I added that function which means the spec should work. But double checking is always sensible.

Reply:
Ah, though there are spec conformers, they don't necessarily get run that I can see. BUT it looks like `aero` actually omits the key entirely if it is just env and isn't set.

Reply:
Dang I wish I knew that earlier.

Reply:
Well I don't see that it's documented, I'd assume the same you did. But the repl does indicate this is the case